examples/arithmetics/README.md

```diff
@@ -0,0 +1,13 @@
+# Arithmetics Example
+
+## Interpreter CLI
+
+The Arithmetics Example features an interpreter that you can run via cli.
+
+Ensure the complete projects was properly built, otherwise run `npm install` from the root of the Langium project.
+Use `node ./bin/cli.js` to run the cli. Follow the instructions or use `node ./bin/cli.js eval <full-path-to-calc-file>`.
```

Comment:
We should rename `cli.js` to `cli` for consistency.

Reply:
Do you want to open a new issue for that?

Reply:
You are allowed to omit the `.js` from files, when invoking them using `node`. So you can omit this from the documentation here as well. Removing the file extension from the file in the repo itself can be done separately.

Reply:
Ok, good to know. We should do the renaming once all examples have CLIs, right?

Reply:
Means: I will only update the documentation for now.

Reply:
Exactly, yes 👍

Reply:
Done, see squashed commit below